### ISSUE TITLE
Interest in a 1080p pack?

### ISSUE THREAD

**Issue by kieraneglin** [quinton-ashley/Super_Mario_Sunshine_UHD_Texture_Pack#13]
Recently, there have been several handheld emulation devices that have enough power to run GCN games with ease (Retroid Pocket 5/mini, etc). These devices are powerful enough to run the game at full 1080p upscaled, but this texture pack provides just slightly too much strain for a reliable 30fps.

To that end, I was wondering if you'd be interested in providing a scaled-down 1080p pack? If so, I'd be happy to create a PR with each image scaled down to 25% of its original size!

Let me know if that sounds like something you'd like to include (:

**Comment by quinton-ashley** [comment]
Yeah! Don't make a PR though cause I don't want to increase the size of the repo. If you want to make a 1080p version of the pack I can add the 7z file to the v2.1.1 release and link to it in the readme.

https://github.com/quinton-ashley/Super_Mario_Sunshine_UHD_Texture_Pack/releases/tag/2.1.1

**Comment by kieraneglin** [comment]
Sure! I'll make a fork then and we can go from there 🤙 

I might need your help in packaging the PNGs into DDS files but I'll give it a shot first!

**Comment by kieraneglin** [comment]
Done! It seems to work well:

![Screenshot_20241224-155413](https://github.com/user-attachments/assets/8cddcd6d-a976-41dc-b81c-fd2d4fdd38b1)

[Here](https://drive.proton.me/urls/0YMKF3YP3C#XQDffgS4cEYq) are the files. I had to upload them elsewhere since GH wouldn't let me push up the zip. I've included a single zip file as well as all the source images. The upload for the source files is still running as of me writing this and it's looking like it'll take up to 45 minutes, but the singular zip file is ready.

There appears to be some alpha channel weirdness in many of the source PNGs that I needed to address since it was causing downscaling to look terrible. The Custom Texture Tool has a utility for this but it takes forever on Windows so I instead used ImageMagick on Linux to repair the PNGs and it worked well! From there, I threw the images back into the Custom Texture Tool and converted them to DDS.

**Comment by kieraneglin** [comment]
Just updating to say that I've been using this for ~2 weeks now and it's been working great!
